RefData/data/10965_add_region_to_venue.sql

```diff
@@ -0,0 +1,397 @@
+SET XACT_ABORT ON
+BEGIN TRANSACTION;
+
+update dbo.HearingVenue set Region = 'LONDON' where VenueCode = '100004';
```

Comment:
could we address the sonar issues for this file?

Reply:
I'm more inclined to just ignore the ref data stuff. I used concat to generate the sql statements from Excel to avoid typoes